agent/bench-scripts/test-bin/require-rpm

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+args="${@}"
+echo "${0} ${args}" >> ${_testlog}
+
+# Mimic behavior of require-rpm by exiting with 0 (for unit tests we always
+# find the RPM) and emitting the RPM version that was found (whatever was given
+# as arguments).
+_rpm="${1}"
+_ver="${2}"
+_match="${3}"
+if [[ ! -z "${_ver}" ]]; then
+    _ver="-${_ver}"
+fi
```

Comment:
There's a succinct idiom for this:
```
_ver=${2:+-${2}}
```
(Use this at line 10 and omit lines 12-14.)